File: DitoSDK/samplejava/src/main/java/br/com/dito/samplejava/DitoApplication.java
package br.com.dito.samplejava;

import android.app.Application;
import br.com.dito.ditosdk.Dito;
import br.com.dito.ditosdk.Options;

public class DitoApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();

        Options options = new Options();
        options.setDebug(true);

        Dito.INSTANCE.init(this, options);
    }
}

File: app/src/main/java/com/example/flutter_sdk_test/DitoApplication.kt
package com.example.flutter_sdk_test

import android.app.Application
import br.com.dito.ditosdk.Dito
import br.com.dito.ditosdk.Options

class DitoApplication : Application() {

    override fun onCreate() {
        super.onCreate()
        val options = Options()
        options.debug = true
        Dito.init(this, options)
    }
}